index.html

```diff
@@ -23,8 +23,8 @@
     <section class="intro-wrapper">
         <section class="welcome">
             <h2 class="greeting-intro">
```

Comment:
Kindly. the main message of your website should be an H1 tag, your website will be more accessible

Reply:
I have adjusted the heading to h1. Thanks for this correction